docs/src/running-validator/validator-start.md

```diff
@@ -306,3 +287,70 @@ the validator to ports 11000-11010.
 
 The `--limit-ledger-size` argument will instruct the validator to only retain the
 last couple hours of ledger. To retain the full ledger, simply remove that arg.
+
+
+### Systemd Unit
+Running the validator as a systemd unit is one easy way to manage running in the
+background.
+
+Assuming you have a user called `sol` on your machine, create the file `/etc/systemd/system/sol.service` with
+the following:
+```
```

Comment:
WDYT about linking out to the cluster repo instead of inlining here?  One less place to keep up-to-date

Reply:
I actually didn't link that on purpose.  Cluster repo is basically internal, but we've been aiming at it since we've not had anything else/better.   And this PR is a baby step towards something else/better :)

Reply:
Ah, gotcha!  I'm sure I'll only forget to bump `LimitNOFILE` here three or four times :wink:

Reply:
📠